pkg/container-utils/cri/cri_test.go

```diff
@@ -0,0 +1,254 @@
+// Copyright 2019-2022 The Inspektor Gadget authors
```

Comment:
I like these tests very much. They are very complete and test many different scenarios. In https://github.com/kinvolk/inspektor-gadget/pull/821 I'm using `reflect.DeepEqual()` and `cmp.Diff()` to avoid having to check each field manually. Maybe you can follow the same approach here? I gave it a try and the result is available here https://github.com/mauriciovasquezbernal/inspektor-gadget/commit/e52a876dab79c595490c81693fae80e0d21d9ba7.